en/snap/integration-guide.md

```diff
@@ -411,55 +437,103 @@ Or try the demo here:
 
 > **Tips:** Optionally, you can also [use Javascript callbacks](/en/snap/advanced-feature.md#javascript-callback) to handle payment events triggered from customer finishing interaction with Snap payment page.
+>
+> <!--Follow the style for Tips, Notes like ?>***Notes***: or ?>***Tips***:-->
 
-## 3. Create Test Payment
+### 3. Creating Test Payment
 
-Create a test payment to make sure you have integrated Snap successfully. You can use one of our test credentials for Card Payment:
+Create a test payment to make sure you have integrated Snap successfully. There are various payment methods available on Snap. You can choose any one of them to create a test payment. Following are the test credentials for Card payment. 
 
 Name | Value
 --- | ---
 Card Number | `4811 1111 1111 1114`
 CVV | `123`
-Exp Month | Any month (e.g: `02`)
-Exp Year | Any future year (e.g: `2025`)
+Exp Month | Any month in MM format. For example, `02`. 
+Exp Year | Any future year, in YYYY format. For example, `2025`. 
 OTP/3DS | `112233`
 
-You can use more test payment credentials [available on our sandbox payment simulator](/en/technical-reference/sandbox-test.md)
+For more details, refer to [Testing Payments on Sandbox](/en/technical-reference/sandbox-test.md).
 
 ![Snap Test Transaction](./../../asset/image/snap-test-transaction.gif)
 
-## 4. Handle After Payment
+### 4. Handling After Payment
+
+When the transaction status changes, Midtrans notifies you at the redirect URL and sends HTTP notification to the merchant backend. This ensures that you are updated of the transaction status securely.
+
+HTTP POST request with JSON body will be sent to your *Payment Notification URL* configured on dashboard.
+
+<details>
+<summary><b>Configuring Payment Notification URL</b></summary>
+<article>
+To configure the Payment Notification URL, follow the steps given below.
+1. Login to your MAP account.
+
+2. On the Home page, go to **SETTINGS > CONFIGURATION**.
+   *Configuration* page is displayed.
 
-Other than customer being redirected, when the status of payment is updated/changed (i.e: payment has been successfully received), Midtrans will send **HTTP Notification** (or webhook) to your server's `Notification Url` (specified on Midtrans Dashboard, under menu **Settings > Configuration `Notification URL`**). Follow this link for more details:
+3. Enter **Payment Notification URL**.
+
+4. Click **Update**.
+   A confirmation message is displayed.
+
+   ![Core API](C:/Users/asset/image/coreapi/core-api-payment-notification-1.png)
 
 <div class="my-card">
 
+ Follow this link for more details:
+
+
 #### [Handling Webhook HTTP Notification](/en/after-payment/http-notification.md)
-Learn how technically your system should handle the Webhook HTTP notification, which will be triggered when transaction status updated.
+Webhook HTTP notification is triggered when transaction status is updated. In this section, you will learn how to handle Webhook HTTP notification. 
 </div>
```

Comment:
This is brokenly rendered

<img width="768" alt="Screen Shot 2020-11-20 at 12 33 39" src="https://user-images.githubusercontent.com/13027142/99763146-b2029080-2b2c-11eb-9e9f-0710ec3dff45.png">